pom.xml

```diff
@@ -1525,13 +1525,26 @@
                     <version>${spotless-plugin.version}</version>
                     <configuration>
                         <java>
-                            <removeUnusedImports />
+                            <removeUnusedImports/>
+                            <!--<forbidWildcardImports /> fix with rewrite: https://docs.openrewrite.org/recipes/java/removeunusedimports -->
                         </java>
+                        <lineEndings/>
+                        <pom>
+                            <sortPom/>
+                        </pom>
+                        <upToDateChecking>
+                            <enabled>true</enabled>
+                        </upToDateChecking>
                     </configuration>
                 </plugin>
             </plugins>
         </pluginManagement>
-
+        <plugins>
+            <plugin>
+                <groupId>com.diffplug.spotless</groupId>
+                <artifactId>spotless-maven-plugin</artifactId>
```

Comment:
is this an issue or can we give ppl to use their IDE without adding some custom maven goal?

<img width="368" height="133" alt="Image" src="https://github.com/user-attachments/assets/24adba42-c88e-4acd-9efc-ca76098a2d58" />